lib/mvi/service.rb

```diff
@@ -1,5 +1,7 @@
 # frozen_string_literal: true
 require 'savon'
+require_relative 'response'
+require_relative 'mock_service'
```

Comment:
I would put that require in development.rb and test.rb possibly as well, the only two places where it is likely to be used, so as to not load a dependency thats not used in production / staging.